app/src/Homepage/Homepage-components.tsx

```diff
@@ -22,10 +43,15 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
 
 
 export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {   
+    const navigate = useNavigate();
     
     const handleClick = () => {
         console.log('Button clicked: ' + classlevel);
```

Comment:
Can you get rid of this print statement? This doesn't seem necessary (though this is not from u)